core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -1247,52 +1283,62 @@ def __get_store_path(self, tns_admin, store):
+        return properties_set
 
     def __set_atp_standard_conn_properties(self, datasource_name, tns_admin, truststore, truststore_pwd,
                                            truststore_type, keystore_pwd, keystore_type, keystore):
         location = deployer_utils.get_jdbc_driver_params_properties_location(datasource_name, self.aliases)
         keystore, keystore_type, truststore, truststore_type = atp_helper.fix_store_type_and_default_value(keystore,
                                                                     keystore_type, truststore, truststore_type)
 
+        properties_set = []
+        self.__set_connection_property_info(location, DRIVER_PARAMS_KEYSTORE_PROPERTY, self.__get_store_path(tns_admin,
+                                        keystore), properties_set)
+
+        self.__set_connection_property_info(location, DRIVER_PARAMS_KEYSTORETYPE_PROPERTY, keystore_type, properties_set)
 
-        self.__set_connection_property(location, DRIVER_PARAMS_KEYSTORE_PROPERTY, self.__get_store_path(tns_admin,
-                                        keystore))
-        self.__set_connection_property(location, DRIVER_PARAMS_KEYSTORETYPE_PROPERTY, keystore_type)
         if keystore_pwd:
-            self.__set_connection_property(location, DRIVER_PARAMS_KEYSTOREPWD_PROPERTY, keystore_pwd, encrypted=True)
-        self.__set_connection_property(location, DRIVER_PARAMS_TRUSTSTORE_PROPERTY, self.__get_store_path(tns_admin,
-                                       truststore))
-        self.__set_connection_property(location, DRIVER_PARAMS_TRUSTSTORETYPE_PROPERTY, truststore_type)
+            self.__set_connection_property_info(location, DRIVER_PARAMS_KEYSTOREPWD_PROPERTY, keystore_pwd,
+                 properties_set, encrypted=True)
+        self.__set_connection_property_info(location, DRIVER_PARAMS_TRUSTSTORE_PROPERTY, self.__get_store_path(tns_admin,
+                                       truststore), properties_set)
+        self.__set_connection_property_info(location, DRIVER_PARAMS_TRUSTSTORETYPE_PROPERTY, truststore_type,
+                                            properties_set)
         if truststore_pwd:
-            self.__set_connection_property(location, DRIVER_PARAMS_TRUSTSTOREPWD_PROPERTY, truststore_pwd,
-                                           encrypted=True)
-        self.__set_connection_property(location, DRIVER_PARAMS_NET_SSL_VERSION, '1.2')
-        self.__set_connection_property(location, DRIVER_PARAMS_NET_SERVER_DN_MATCH_PROPERTY, 'true')
-        self.__set_connection_property(location, DRIVER_PARAMS_NET_TNS_ADMIN, tns_admin)
-        self.__set_connection_property(location, DRIVER_PARAMS_NET_FAN_ENABLED, 'false')
+            self.__set_connection_property_info(location, DRIVER_PARAMS_TRUSTSTOREPWD_PROPERTY, truststore_pwd,
+                 properties_set, encrypted=True)
+        self.__set_connection_property_info(location, DRIVER_PARAMS_NET_SSL_VERSION, '1.2', properties_set)
```

Comment:
Maybe the SSL version value should be a constant since presumably, it may change in the future?